man/meson.build

```diff
@@ -198,16 +198,14 @@ run_target(
 
 ############################################################
 
-if dbus_docs.length() > 0
```

Comment:
If `xsltproc` is not found, then `dbus_docs` is empty.

Reply:
Ooops. Apparently there was a reason for this conditional. I removed that commit now.